decidim-bulletin_board-js/src/client/operations/get_log_entry.gql

```diff
@@ -0,0 +1,9 @@
+query GetElectionLogEntries($electionUniqueId: String!, $contentHash: String!) {
+  logEntry(electionUniqueId: $electionUniqueId, contentHash: $contentHash) {
+    logEntry {
```

Comment:
Is this correct? Looks a bit odd that you have two `logEntry`. I would expect the query to be like this:
```graphql
query {
  logEntry(...) {
    messageId
    signedData
  }
}
```

Otherwise the shape of your response will be `data.logEntry.logEntry.messageId`